modules/security-context-constraints-about.adoc

```diff
@@ -8,6 +8,11 @@
 
 Similar to the way that RBAC resources control user access, administrators can use _security context constraints_ (SCCs) to control permissions for pods. These permissions include actions that a pod can perform and what resources it can access. You can use SCCs to define a set of conditions that a pod must run with to be accepted into the system.
 
+[IMPORTANT]
+====
+There is a difference in permissions experienced by containers, when you create pods directly by `kubectl` or `oc` and when pods are being created by some higher level type like deployment or job. The pods created by higher level resources use secured `ServiceAccount` access, but creating the pod uses `kubectl` or `oc` user access, which may exceed safe {product-title} standards. The set of SCCs that admission uses to authorize a pod are determined by the user identity and groups that the user belongs to. Additionally, if the pod specifies a service account, the set of allowable SCCs includes any constraints accessible to the service account.
+====
+
```

Comment:
This information is already present in https://docs.openshift.com/container-platform/4.10/authentication/managing-security-context-constraints.html#admission_configuring-internal-oauth. Possibly outline it there.

Reply:
I do not find this explanation of this discrepancy on the page you've linked.

This is confusing, so I'd say having it here in plain text like this is beneficial, as people run pods using `kubeadmin` account. Of course, we could link to https://docs.openshift.com/container-platform/4.10/authentication/managing-security-context-constraints.html for more details.

Reply:
@stlaz @cardil so should we keep the note as is?

Reply:
Quoting the page I linked earlier
> The set of SCCs that admission uses to authorize a pod are determined by the user identity and groups that the user belongs to. Additionally, if the pod specifies a service account, the set of allowable SCCs includes any constraints accessible to the service account.

I agree that it is confusing and does not lead directly to the conclusions made by @xenolinux here.

I would propose to move this note somewhere to the explanation quoted earlier.

However, I would also ask to word the new note a bit better. Specifically:
> There is a difference in permissions experienced by containers

I don't understand this sentence. Containers don't "experience" permissions.

> higher level type

Deployment is not a type. You can call these "Pod controllers" if you would.

> use secured `ServiceAccount` access

I don't understand this part, how is this access secured?

> but creating the pod uses `kubectl` or `oc` user access

You can create the pod directly even without kubectl or oc. You can create the pod using a controller with kubectl or oc. I propose to use "create directly".

> which may exceed safe {product-title} standards.

I would leave this out, it's misleading

> The set of SCCs that admission uses to authorize a pod are determined by the user identity and groups that the user belongs to. Additionally, if the pod specifies a service account, the set of allowable SCCs includes any constraints accessible to the service account.

This is a copy-paste of the paragraph I was pointing to originally, by moving this note to my proposed spot you can remove this part.

Reply:
Addressed